apps/nuxt3-ssr/components/harmonization/HarmonizationTableCellAvailableIcon.vue

```diff
@@ -0,0 +1,25 @@
+<script setup lang="ts">
+import type { HarmonizationStatus } from "~/interfaces/types";
+
+const props = defineProps<{
+  status: HarmonizationStatus;
+}>();
+
+const availableStatuses = ["partial", "complete"];
+
+const recodedStatus = computed(() => {
+  if (availableStatuses.includes(props.status)) {
+    return "available";
+  }
+  return props.status;
+});
+</script>
+<template>
+  <div
+    :data-status="recodedStatus"
```

Comment:
whats the dataStatus ?

Reply:
This line was added to eliminate the computed values for css classes. This line creates a `data-*` attribute and binds the `status` to it. The icon is then styled using data attribute selector rather computed variables.